packages/synthetics-sdk-broken-links/test/unit/broken_links.spec.ts

```diff
@@ -299,97 +381,10 @@ describe('runBrokenLinks', async () => {
       '/example_html_files/200.html',
       '/example_html_files/file_doesnt_exist.html',
     ];
-    broken_links_result?.followed_link_results?.forEach((link, index) => {
-      expect(link.target_uri.endsWith(expectedTargeturis[index]));
-    });
+    broken_links_result?.followed_link_results?.forEach(
+      (link: BrokenLinksResultV1_SyntheticLinkResult, index: number) => {
+        expect(link.target_uri.endsWith(expectedTargeturis[index]));
+      }
+    );
   }).timeout(150000);
-
-  it('Completes a full passing execution', async () => {
```

Comment:
Maybe I missed this getting moved somewhere else, is there a reason this test got deleted?